Car/src/components/journey-new-applicant/journey-new-applicant-view/JourneyNewApplicantView.tsx

```diff
@@ -66,6 +67,13 @@ const JourneyNewApplicantView = (props: JourneyNewApplicantViewProps) => {
         });
     }, []);
 
+    useEffect(() => {
```

Comment:
It is better to use the request result in line 58 to get departure time and set JourneyIsFinished than send a new request.

Reply:
Changed